### ISSUE TITLE
Add focus styling to all interactive elements

### ISSUE THREAD

**Issue by mgifford** [wet-boew/wet-boew#8838]
I've been promoting this for a long time in the Drupal community:
  https://www.drupal.org/project/drupal/issues/1993574

Noticed that the UK is doing a good job of highlighting keyboard only focus:
https://www.gov.uk/guidance/make-your-website-or-app-accessible-and-publish-an-accessibility-statement

<img width="651" alt="Screenshot of the UK's keyboard only focus" src="https://user-images.githubusercontent.com/116832/76525179-c04ac980-6441-11ea-9cbe-b0b82e5ca3d0.png">

Why isn't this part of WET?

**Comment by duboisp** [comment]
I am fine with this suggestion.

What do you think @bsouster, @delisma?

**Comment by bsouster** [comment]
Coincidentally @delisma and I were talking about doing exactly this two weeks ago at ISED codesprint. 

I haven't had a chance to yet and maybe I'm jumping ahead when this is just a draft, but I suggest we check that it passes the WCAG 2.2 success criteria (Focus visible (enhance))[https://www.w3.org/TR/WCAG22/#focus-visible-enhanced] so we only do this once

**Comment by bsouster** [comment]
@duboisp is there someone we should include to make sure new focus highlight change agrees with the GoC's colour palette? Or someone on @delisma's side?

**Comment by GormFrank** [comment]
@delisma What is your input on that?

**Comment by delisma** [comment]
@mgifford @bsouster This should be flagged in GCWeb not WET-BOEW IMHO.

**Comment by bsouster** [comment]
@GormFrank sounds like you're giving me blank permission to do whatever I want for WET-BOEW  :)

I will do something for WET and leave it open to discussion, but UX testing should be an important part of the final decision. For WET I am fine to use what UK clearly decided gave the best results. DTO should be involved for GCWeb with respect to brand compatible colours.

**Comment by JuliannaR** [comment]
Curious on the thoughts here around the contrast of the yellow keyboard highlight and white when this treatment is given in text links in content. Been seeing that a lot lately, but without the thick line. BBC has some interesting changes the UX of their design patterns around this that work on both light and dark themes.

**Comment by mgifford** [comment]
Ultimately, I think it would be interesting to test a pattern and do some usability work around it. I'm not a huge fan of the bright yellow, although it certainly gets your attention.  Ideally there would be a greater level of personalization that would allow people to have greater choice over how focus is displayed.

**Comment by JuliannaR** [comment]
Been exploring a pattern like this which has an inner and outer colour, works on light and dark themes and has a clear bounding box.

![Screen Shot 2021-05-04 at 2 43 35 PM](https://user-images.githubusercontent.com/35537308/117316728-d93ce800-ae56-11eb-9f80-3246d5559b9a.png)
